src/routes/settings/+page.svelte

```diff
@@ -86,7 +108,12 @@ $: {
       <label>Name: <input bind:value={name}></label><br>
       <label>Avatar: <input bind:value={avatar}></label><br>
       <img class="profile-img" src={avatar} alt='profile'><br>
-      <label>nip05: <input bind:value={nip05}></label><br>
+      <label>
+        nip05: 
+        <input bind:value={nip05}>
+        <span>{nip05Status || ''}</span>
```

Comment:
nip05Status never is undefined, so it always shows one of the three texts. On an empty string it shows "valid". That's a bit weird.

Reply:
Do you want to allow empty nip05 but not show "valid"?